### PR TITLE
xfreerdp: add /drive example

### PR DESCRIPTION
**Pull request by cyberimpulse**
Added command to map a shared drive, enables upload or download of files between local and remote hosts.

- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/xfreerdp.md [hunk 1]
```diff
@@ -22,3 +22,7 @@
 - Connect to a FreeRDP server ignoring any certificate checks:
 
 `xfreerdp /v:{{ip_address}} /u:{{username}} /p:{{password}} /cert:ignore`
+
+- Connect to a FreeRDP server with shared drive mapped:
+
+`xfreerdp /v:ip_address /u:username /p:password /drive:{{path/to/directory}},{{share_name}}`
```
Comment (kbdharun) [inline pages/linux/xfreerdp.md:28]:
```suggestion
`xfreerdp /v:{{ip_address}} /u:{{username}} /p:{{password}} /drive:{{path/to/directory}},{{share_name}}`
```
Thanks for your contribution and welcome to tldr. We use `{{}}` syntax for our clients to highlight placeholders in order to distinguish them from the rest of the command.

image

Comment (cyberimpulse) [reply pages/linux/xfreerdp.md]: Thank you. I should have included the {{}} syntax for all of the placeholders

**Review by kbdharun — APPROVED** [review]

File: pages/linux/xfreerdp.md [hunk 2]
```diff
@@ -22,3 +22,7 @@
 - Connect to a FreeRDP server ignoring any certificate checks:
 
 `xfreerdp /v:{{ip_address}} /u:{{username}} /p:{{password}} /cert:ignore`
+
+- Connect to a FreeRDP server with shared drive mapped:
```
Comment (acuteenvy) [inline pages/linux/xfreerdp.md:26]:
```suggestion
- Connect to a FreeRDP server with a shared directory:
```

File: pages/linux/xfreerdp.md
Comment (cyberimpulse) [inline pages/linux/xfreerdp.md:1]: Thank you, that’s more clearly worded

File: pages/linux/xfreerdp.md (hunk 2 above)
Comment (cyberimpulse) [reply pages/linux/xfreerdp.md]: Thank you, that’s more clearly worded

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**